### PR TITLE
Addition of Standard Effective Temperature (SET) as an Output Variable

### PR DESCRIPTION
**Pull request by RKStrand**
Pull request overview
---------------------
Based on discussion on unmethours.com, there was a request to add Standard Effective Temperature (SET) as an output variable.  This is calculated as part of the Pierce two-node thermal comfort model.  This work fulfills this request and it is now possible to select SET as an output variable in EnergyPlus.  This includes code changes, a unit test, documentation updates, and a modified test suite input file to show the use of this new output variable.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue #7174 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes n/a
 - [x] Required documentation updates?

### CODE REVIEW

File: tst/EnergyPlus/unit/ThermalComfort.unit.cc
Comment (mjwitte) [inline tst/EnergyPlus/unit/ThermalComfort.unit.cc]: Would prefer a much more lightweight unit test. Seems you could just have zone, people, and schedules in the idf snippet and then just call the functions to process zone and people data and then CalcThermalComfortPierce.

File: src/EnergyPlus/ThermalComfort.cc
Comment (mjwitte) [inline src/EnergyPlus/ThermalComfort.cc]: OutputProcessor::Unit::C ??

File: tst/EnergyPlus/unit/ThermalComfort.unit.cc
Comment (mjwitte) [inline tst/EnergyPlus/unit/ThermalComfort.unit.cc]: And when you merge in develop, this line will fail. Simply delete it.

Comment (RKStrand) [reply tst/EnergyPlus/unit/ThermalComfort.unit.cc]: Will delete it.

File: tst/EnergyPlus/unit/ThermalComfort.unit.cc
Comment (mjwitte) [reply tst/EnergyPlus/unit/ThermalComfort.unit.cc]: Moderately important. This should get your schedule in place, if done before processing the people:
```
    ScheduleManager::ProcessScheduleInput();
    ScheduleManager::ScheduleInputProcessed = true;
```

Comment (RKStrand) [reply tst/EnergyPlus/unit/ThermalComfort.unit.cc]: @mjwitte Ok, it's now much more lightweight.  Doesn't call anything except the Pierce subroutine.  Sets every manually in the unit test now.  This should be up in GitHub as of now.

**Comment by rraustad** [comment]
New report variable added but no changes to table reports.

**Comment by rraustad** [comment]
- Final code and unit test review, removed clear_state access from unit test
 - Compiled and reviewed IO Ref., small edits, no new figures to add to CMake.
 - Ran sequential unit tests in release mode without failure
 - Ran example file, no warnings in err file. Same for annual run.
 - Verified new report variable added to existing example file (TermReheatSurfTC.idf)
 - Pulled develop and uploaded final changes
 - Plotted new report variable for design days

image

**Review by rraustad — APPROVED** [review]
Minor editorial mods to IO Ref, correct unit test, uploaded changes.

**Comment by rraustad** [comment]
This is ready to merge once CI has a chance to process changes.

**Comment by rraustad** [comment]
@RKStrand can you comment on the SET being that much above the summer design day zone temp in the figure above (e.g., occupants have on too many clothes, etc.)? The summer zone humidity ratio is actually lower than winter and zone temp is not that much different. No other code changes will be requested, just wondering if this is working as expected (i.e., if new issue should be opened).

**Comment by RKStrand** [comment]
@rraustad I see what you are saying in the results, but I think this is actually what one would expect.  SET includes both radiative and convective aspects as well as a bunch of other things.  I would expect that SET would exhibit somewhat similar behavior to MAT but the surface temperatures in the space will dampen the response and make SET more closely related to outside air temperature (since that will in turn affect inside surface temperatures).  So, I think that what you are seeing is actually correct--SET is lower than MAT for the most part for winter (because MRT is lower as a result of the cold exterior conditions) and SET is generally higher than MAT for summer.  The "heavy" surfaces are damping the response of SET so when MAT changes rapidly like it does at the end of the work day in the summer data MAT can flip so that it is higher than SET because MAT in that situation is actually higher temporarily for that time.  Basically, if one were to graph MAT, MRT, and SET, I would guess that in general SET should track between these two for the most part.  Hopefully that makes sense, but if it doesn't, please let me know.  Overall, I think what you are seeing is the right trend and I don't think there is a problem here that needs to be logged as a defect.  Thanks for asking and for diligently checking results to see if there are additional problems!

**Comment by rraustad** [comment]
Followup graph based on results discussion.

image

**Comment by rraustad** [comment]
I can't think of a reason why this should not merge except for a complete CI assessment...

**Comment by RKStrand** [comment]
@Myoldmopar This has met with approval from @rraustad but it sounds like there wasn't a complete CI result?  Does it need to be relaunched or something?

**Comment by Myoldmopar** [comment]
The CI machine itself needed to be relaunched.  And it has been done.  CI results should be coming in soon.

**Comment by rraustad** [comment]
Diff's from Coil Sizing report fixes #7435 are creeping in here. Merged that one into develop 17 hrs ago.

**Comment by rraustad** [comment]
Well, I was just going to merge this anyway since I understood the diff's. But it can't hurt to have a fresh clean CI result.

**Comment by RKStrand** [comment]
@rraustad Oops--sorry to get in the way.  Should have left it alone.  But as you pointed out, a fresh CI can't (shouldn't) hurt.

**Comment by Myoldmopar** [comment]
We've had an issue with the CI box and had to take it down.  I'm working to get some extra support stood back up but this is likely to need to wait until after the weekend.

**Comment by rraustad** [comment]
CI machines now match with expected results.

**Pull request merged**
